### PR TITLE
Fix: link on /hero

### PR DESCRIPTION
**Pull request by diwash007**
Hey folks,
Can you guys test it out on your end?
I got no idea about Rust, but my mind told me this might be the solution

### CODE REVIEW

**Comment by kymppi** [comment]
Looks about right. I can test it in few hours 😃

**Review by washbin — APPROVED** [review]
Looks Good To Me

**Review by washbin — CHANGES_REQUESTED** [review]
Please format appropriately using cargo fmt :smile:

File: src/commands/team.rs [hunk 1]
```diff
@@ -118,11 +118,11 @@ pub async fn run(ctx: Context, command: ApplicationCommandInteraction) {
 
                 if all_prs[0..2].len() == index {
                     pull_req_cloned +=
-                        &format!("<:reply:1029065416905076808>[{}]({})\n", pr.title, pr.url);
```
Comment (washbin) [inline src/commands/team.rs]: seems like you replaced the url fields on team.rs as well, this uses a different api
This file is probably better left untouched for issues related to heroes :smile: .
removing the changes from this file should probably fix the failing test

**Review by thenishantsapkota — APPROVED** [review]

**Review by washbin — CHANGES_REQUESTED** [review]
Please address the issue with team.rs

**Review by washbin — COMMENTED** [review]
looks great to me now. let's see what the workflows say

**Review by washbin — APPROVED** [review]

**Comment by diwash007** [comment]
yay, passed them all
Time for electrolysis

**Pull request merged**